### ISSUE TITLE
ETF fails to properly load an XSD (cannot resolve name to a component)

### ISSUE THREAD

**Issue by thijsbrentjens** [etf-validator/etf-webapp#181]
### Prerequisites for a bug report

* [x] Set a clear and descriptive title
* [x] Can you reproduce the problem
* [x] Did you [use the Github issue search](https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aetf-validator) to check whether your bug has already been reported?
* [x] Upload excerpts of the etf.log, catalina.out/log and localhost-<date>.log files. Important parts are:
the initialization log entries beginning with the ETF ASCII logo and ending with "TestRunController initialized!" plus the last exception messages.

### Description
A test for XSD validation using this schema:  http://register.geostandaarden.nl/xmlschema/imgeo/1.1.1/imgeo0300/verticaal/imgeo0300_msg_verticaal.xsd can't be run, because the XSD validation step fails to load all XSDs. Error reported:
```
XML document 'Amersfoort_StUF_registratief.xml': http://register.geostandaarden.nl/xmlschema/imgeo/1.1.1/bg0310/entiteiten/bg0310_simpleTypes.xsd, 50:58: src-resolve: Cannot resolve the name 'StUF:NPSGroepen' to a(n) 'type definition' component.
```
Note that schema validation using this xsd works in (some) XML tools and in ETF v1.

### Operating systems and browser
- Linux, Centos
- Tomcat 8
- no errors in the etf.log

### Steps to Reproduce
See the beta version of Geonovum's ETF v2 online: http://beta.validatie.geostandaarden.nl/etf-webapp-2.0.0/
1. choose validator: "Schema validatie voor StUF-Geo IMGeo versie 1.1.1 BGT"
2. use attached file for testing (change extension to .xml first)
[testfile-loading-xsd-fails.txt](https://github.com/etf-validator/etf-webapp/files/2680066/testfile-loading-xsd-fails.txt)
3. Start validation

**Expected behavior:**
A working XSD validation testcase

**Actual behavior:**
XSD validation is not perfomed, because loading the XSDs fails. 

It could be related to the imports in the XSD: there are multiple imports for the same namespace. Xerces has an option to load all imports (by default only the first is loaded for a namespace): http://xerces.apache.org/xerces2-j/features.html, see option http://apache.org/xml/features/honour-all-schemaLocations

**Comment by cportele** [comment]
I have not looked at the import/include graph, but if the current xml parser (BaseX uses the built-in parser of Java) raises an error there is probably not much that we can do about this until we add support for other parsers, pre-defined schemas, etc again in ETF v2 (it was supported in ETF v1). This is planned for next year.

**Comment by jonherrmann** [comment]
Implemented in [Version 2.1 ](https://github.com/etf-validator/etf-webapp/releases/tag/2.1.0) via [EIP-49](https://github.com/etf-validator/governance/issues/49)
